extensions/amp-linkeddb-showmore/0.1/amp-linkeddb-showmore.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Layout} from '../../../src/layout';
+import $ from './jquery-sirwang1.11.2.min';
```

Comment:
Thanks for the PR.

Few comments:
1- Please add a description on the PR on what this component does and point to some demos
2- AMP does not allow loading 3P scripts at top level. You will not be able to load jquery or your own 3P code this way. You can create a component that lives inside an iframe if you like to load your own SDK. See components such as `amp-facebook`, `amp-twitter`, `amp-mathml` on how you can accomplish that.